plugin/storage/kafka/options.go

```diff
@@ -24,24 +24,18 @@ import (
 )
 
 const (
-	configPrefix   = "kafka"
-	suffixBrokers  = ".brokers"
-	suffixTopic    = ".topic"
-	suffixEncoding = ".encoding"
```

Comment:
not sure why you're removing the encoding param - we may only support one value at this time, but so what...